src/util/duration.h

```diff
@@ -136,19 +157,19 @@ class Duration {
     }
 
     QString formatNanosWithUnit() const {
-        return QString("%1ns").arg(toNanos());
+        return QString("%1ns").arg(toIntegerNanos());
```

Comment:
MIDI timestamps are the case where saving 1 space is useful. You said so on #815 which is why I removed the space in the first place: 

> Do you a way to keep the controllerDebug output below 80 columns? 

Personally, I find it easier to read when there is no space. The unit attached to the number somehow clarifies for me what that number means without having to read the next token in the string.

Also, I realize this is something you care about and I recognize that -- but I didn't touch the format methods in this PR and am not feeling motivated to change them in this PR when a) I didn't touch this code and b) I don't agree with your preference.